Dockerfile.template

```diff
@@ -4,7 +4,7 @@ RUN apt-get update && apt-get install -y curl wget build-essential libelf-dev aw
 COPY . /usr/src/app
 WORKDIR /usr/src/app
 
-ENV VERSION '2.29.0+rev1.prod'
-RUN ./build.sh %%BALENA_MACHINE_NAME%% $VERSION example_module
+ENV VERSION '2.29.0+rev1.prod 2.48.0+rev3.prod'
+RUN ./build.sh --device %%BALENA_MACHINE_NAME%% --os-version $VERSION --src example_module
```

Comment:
i think this needs to be `RUN ./build.sh build ...`

Reply:
Fixed